frontend/src/scenes/billing/v2/test/Billing.tsx

```diff
@@ -60,8 +52,7 @@ export function BillingV2({ redirectPath = '' }: BillingV2Props): JSX.Element {
         </Link>
     )
 
-    const products =
-        enterprisePackage && billing?.products_enterprise ? billing?.products_enterprise : billing?.products
+    const products = billing?.products_enterprise ? billing?.products_enterprise : billing?.products
```

Comment:
@kappa90 I am very confused about what was happening here. It seems like if `enterprisePackage` was toggled to `true`, and `products_enterprise` existed, then we'd use those enterprise products. The problem is that in line 41 above, we always set `enterprisePackage` to `false` and the only time it got changed was when someone manually toggled the toggle. What was the intention with that?

The `products` var is used later to show the gauges for the different products. How can we tell if someone is on an Enterprise plan without using this toggle (that I have now deleted)?

Reply:
This toggle would show users an additional Enterprise plan, which would add up to the pricing breakdown to the right, so that they would understand the additional charge, and it would then toggle between `standard` and `enterprise` plans when clicking `Setup Payment`.

If you removed the flag like you did, we would always show the available enterprise plans. I fixed it so that we only show non-enterprise plan if not subscribed (since we use them on the usage breakdown)